app/renderer/settings.js

```diff
@@ -46,6 +46,7 @@ const {
     updateGuiVisibility,
     getMouseConf
 } = require("./common")
+const {setBookmarkSettings} = require("./bookmarks")
```

Comment:
Please import this as needed when used, please only import `common`, `util` and packages at the top (outside of very specific cases). This is faster and uses less memory when bookmarks are not accessed.